xline/src/main.rs

```diff
@@ -110,19 +110,12 @@
     clippy::multiple_crate_versions, // caused by the dependency, can't be fixed
 )]
 
-use std::net::{AddrParseError, SocketAddr};
-
 use anyhow::Result;
 use clap::Parser;
 use log::debug;
+use std::net::SocketAddr;
```

Comment:
std need to be put before third party libs

Reply:
cargo fmt will sort them alphabetically,  this modification will make format CI fail

Reply:
You need to add a empty line after std, please check other files